Cubical/Categories/Instances/PosetCat/Monotone.agda

```diff
@@ -0,0 +1,207 @@
+{-# OPTIONS --safe #-}
+
+-- TODO: This could be generalized to handle monotone functions on
+-- both preorders and posets.
+
+module Cubical.Categories.Instances.PosetCat.Monotone where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Function hiding (_$_)
+open import Cubical.Foundations.Structure
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Relation.Binary.Base
+open import Cubical.Reflection.Base
+open import Cubical.Reflection.RecordEquiv
+open import Cubical.Data.Sigma
+
+open import Cubical.Relation.Binary.Poset
+
+open BinaryRelation
+
+
+private
+  variable
+    ℓ ℓ' : Level
+
+
+
+-- If Y is a set, then functions into Y form a set
+isSetArrow : {X Y : Type} -> isSet Y -> isSet (X -> Y)
```

Comment:
This is in the stdlib already, it's called `isSetΠ`

Reply:
removed.